core/test/lib/dependency-graph/simulator/simulator-test.js

```diff
@@ -202,10 +202,10 @@ describe('DependencyGraph/Simulator', () => {
     });
 
     it('should not reuse connections', () => {
-      const nodeA = new NetworkNode(request({startTime: 0, endTime: 1}));
-      const nodeB = new NetworkNode(request({startTime: 2, endTime: 3}));
-      const nodeC = new NetworkNode(request({startTime: 2, endTime: 5}));
-      const nodeD = new NetworkNode(request({startTime: 2, endTime: 7}));
+      const nodeA = new NetworkNode(request({networkRequestTime: 0, networkEndTime: 1}));
```

Comment:
i noticed there's a bunch of other `startTime` and `endTime`'s in this file.. and while some belong to simulator nodes.. i think many should get renamed too..

Reply:
I'll rename these in the followup lantern change.

Reply:
hold up how the hell are these other tests passing?!

like `should simulate cached network graphs`, the start/end time on nodeA/B just don't seem to matter (delete them, still passes), even the timing assert still passes 😕

Reply:
it's b/c for that test the timing data comes from an estimate based on disk cache, real timing data is ignored. 
<img width="648" alt="image" src="https://user-images.githubusercontent.com/4071474/200074482-6f88b79b-e7da-4c26-b5c1-10494a362279.png">

Reply:
I didn't like how simulator naming got more complicated because of network timing internals, so I kept the start/end time naming in these test files and just modified the `request()` helper function to translate back to the network domain